app/retail/templates/shared/head.html

```diff
@@ -60,7 +60,8 @@
 <title>{% if title %}{{title}}{% else %}{% trans "Grow Open Source" %}{% endif %} | Gitcoin</title>
 {% include 'shared/favicon.html' %}
 <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no" />
-
+{% if meta_title %}<meta name="title" content="{{meta_title}}" />{% endif %}
```

Comment:
Won't doing this collide with all the other pages which have meta title mentioned in the html? Those pages have two meta title tags. Multiple meta tags of same kind are often penalised by search engines for spamming and is not a good practice. I am not sure what the solution is other than providing it from view for all the pages.

Reply:
I don't think we would have places where both `meta_title` is passed from the views.py as well as defined in the html page.
This would a decent start and one of us can shift the rest as a part of tech-debt right ?

Reply:
I set up the if statements in the template here to avoid that since this is a commonly shared template - my branch is the only branch that uses `meta_title` in the context so we should be good 👍